src/neural/network_opencl.cc

```diff
@@ -16,7 +16,9 @@
  along with Leela Chess.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-#include "neural/CL/transforms.h"
+#include "BLAS/blas.h"
```

Comment:
`neural/BLAS/blas.h`

And maybe still `blas` and `opencl` instead of `BLAS` and `OPENCL`. But if you don't want to do that, I can do that myself later.